docs/DeveloperGuide.adoc

```diff
@@ -714,42 +714,127 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
 See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-by-step solution.
 
 [appendix]
+== Target Users
+
+This application is targeted towards University students, aiming to meet their needs throughout their university life, and hopefully beyond. University students meet many people in the university, such as project group members, tutors, lecturers, friends and other connections. Nowadays it is important for students to build connections with other people, as connections may also help students for their future career after graduation. Since the students meet a lot of people, they may not be able to remember all the information about the people they met. Therefore, iContacts aims to help students to store and manage their contacts easily.
+
+*  User Profile
+
+**  University Student
+**  Student part of many CCA clubs and committees
+**  Student taking role of Teaching Assistant/Tutor
+**  Student who is interning and has colleagues
+**  Student having close friends and family
+
+[appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
+|`* * *` |busy user |search using accumulative link:@attributes[attributes] |narrow down my search in the shortest time possible
+
+|`* * *` |careless user that finds it troublesome to follow a particular order of entering link:#contact-information[contact information] |add contact information without following any particular order |enter contact information more easily and conveniently
+
+|`* * *` |forgetful user that can't remember all link:#tag[tags] |view a list of existing tags |recall and reuse the tags associated with a person
+
+|`* * *` |organized user |search for contacts by a given tag |get the list of contacts with the same tag
+
+|`* * *` |sociable user that was given contacts by another person |import from another source file |add contacts that were shared with me
+
+|`* * *` |sociable student |export selected contacts |share my contacts with friends
+
+|`* * *` |sociable user |store the person's birthday |keep track of the person's birthday
+
+|`* * *` |user |sort the list by name alphabetically |view contacts in an alphabetical order
+
+|`* * *` |user with limited details of a contact |add user with just the name and phone number |store contact information with limited detail
```

Comment:
may be reword this? you don't specify what limited details you have, but want to add name and phone number

Reply:
made the changes. Limited details include name and phone number